Integrations/integration-XFE.yml

```diff
@@ -525,21 +569,28 @@ script:
       description: For malicious URLs, the reason for the vendor to make the decision
     - contextPath: URL.Malicious.Score
```

Comment:
lets remove the `URL` outputs from here

Reply:
won't this break backwards compatibility?

Reply:
no, because its just declaration of outputs, we are still outputting these in the code
just need to make sure we are actually outputting them in the code

Reply:
the bc checks use the context output paths in the yml to check so this change is failing the pre-commit hooks... do  you want me to commit without running the pre-commit hooks?

Reply:
yes please, we will trigger a build to run without bc checks